Common/src/main/java/com/blamejared/crafttweaker/api/zencode/impl/loader/ScriptRun.java

```diff
@@ -101,127 +89,81 @@ public void run() throws Exception {
         }
         
         initializeBep();
-        registerModules();
+        configureRun();
         readAndExecuteScripts();
-        
     }
     
     private void initializeBep() {
         
         this.bep = new IgnorePrefixCasingBracketParser();
-        Services.EVENT.fireRegisterBEPEvent(bep);
-        
-        final List<Class<? extends IRecipeManager>> recipeManagers = CraftTweakerRegistry.getRecipeManagers();
-        bep.register("recipetype", new RecipeTypeBracketHandler(recipeManagers));
-        bep.register("constant", new EnumConstantBracketHandler());
-        
+        Services.EVENT.fireCustomRegisterBepEvent(this.scriptLoadingOptions.getLoaderName(), this.bep::register);
```

Comment:
For being future-proof should we just send the whole scriptLoadingOptions object?
Is there anything in it that others may distinct on?
Like, special handling if it formats/executes or depending on the LoaderSource?

Though, on second thought it does sound like YAGNI, what do you think 🤔